src/garage/sampler/is_sampler.py

```diff
@@ -172,11 +172,11 @@ def sample_isweighted_paths(
             path['rewards'] *= is_ratio
```

Comment:
Is this used at all?

Reply:
Depending on algorithms it use I think yes.

Reply:
Oh yes it's passed by ref